### ISSUE TITLE
Document Static Middleware

### ISSUE THREAD

**Issue by McSneaky** [adonisjs/docs#371]
Currently `'Adonis/Middleware/Static',` is documented only under `middelware` section of documentation. https://adonisjs.com/docs/4.1/middleware#_server_middleware

I think it should be also noted under [file uploads](https://adonisjs.com/docs/4.1/file-uploads) or [file storage](https://adonisjs.com/docs/4.1/file-system) sections.

It is really inconvenient and little bit illogical that you need to read through middleware section of documentation too if you want to receive files from public folder.
In middleware part of documentation it is only mentioned once in example, but there is nothing to explain this line / middleware

I can fill in PR if anyone has ideas where it should be documented. I'm not sure myself, if it should be somewhere under file uploads or file storage 🤔

**Comment by thetutlage** [comment]
I don't think, this should really go under `file uploads`, since I do not recommend uploading files to `public` folder directly.

I believe having a new document that talks about `public files` and mentioning the middleware there is a good choice. What you think?

**Comment by McSneaky** [comment]
I'd also like to +1 Romains point ([in Discord](https://discordapp.com/channels/423256550564691970/423263695804563456/615278321856544789)), that should suggest using CDN / proxy like Nginx etc to serve static files somewhere in documentation.

>I believe having a new document that talks about public files and mentioning the middleware there is a good choice. What you think?

Seems like good idea 👍

**Comment by thetutlage** [comment]
Yup, happy to recommend that. Mind doing a PR with the initial doc?

**Comment by McSneaky** [comment]
Sure, will file it in after work
